svelte/bootstrap/src/components/select/Select.svelte

```diff
@@ -33,44 +33,27 @@
 	});
 	export const api = widget.api;
 	const {
-		stores: {
-			id$,
-			ariaLabel$,
-			badgeClassName$,
-			className$,
-			filterText$,
-			highlighted$,
-			menuClassName$,
-			menuItemClassName$,
-			open$,
-			placement$,
-			selectedContexts$,
-			slotBadgeLabel$,
-			slotItem$,
-			visibleItems$,
-		},
+		stores: {highlighted$, open$, selectedContexts$, slotBadgeLabel$, slotItem$, visibleItems$, className$, filterText$},
 		state$,
-		actions: {onInput, onInputKeydown, onBadgeKeydown},
-		directives: {floatingDirective, hasFocusDirective, referenceDirective, inputContainerDirective},
+		directives: {
+			floatingDirective,
+			hasFocusDirective,
+			referenceDirective,
+			inputContainerDirective,
+			badgeAttributesDirective,
+			inputAttributesDirective,
+			menuAttributesDirective,
+			itemAttributesDirective,
+		},
 	} = widget;
 
 	$: widget.patchChangedProps($$props);
-	$: menuId = `${$id$}-menu`;
 </script>
 
 <div use:referenceDirective class="au-select dropdown border border-1 p-1 mb-3 d-block {$className$}">
-	<div
-		use:hasFocusDirective
-		use:inputContainerDirective
-		role="combobox"
-		class="d-flex align-items-center flex-wrap"
-		aria-controls={menuId}
-		aria-haspopup="listbox"
-		aria-expanded={$open$}
-	>
+	<div use:hasFocusDirective use:inputContainerDirective class="d-flex align-items-center flex-wrap">
```

Comment:
Please add the `{...ssrAttributes({directive})}` to all directives.

Reply:
Never mind, David will start working on the preprocessor soon, you can forget about this comment.